pontoon/base/templates/header.html

```diff
@@ -39,8 +39,10 @@
             <ul>
               {{ Profile.top_menu() }}
 
-              <li><a href="{{ url('pontoon.terms') }}"><i class="fa fa-gavel fa-fw"></i>Terms of Use</a></li>
-              <li><a href="https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/"><i class="fa fa-life-ring fa-fw"></i>Help</a></li>
+              <li><a href="{{ url('pontoon.terms') }}" rel="noopener noreferrer" target="_blank"><i class="fa fa-gavel fa-fw"></i>Terms of Use</a></li>
+              <li><a href="https://github.com/mozilla/pontoon/" rel="noopener noreferrer" target="_blank"><i class="fab fa-github fab-gh"></i>Hack it on GitHub</a></li>
```

Comment:
Instead of adding a custom `fab-gh` class, you should add `fa-fw`, which other icons use to set the [fixed width](https://fontawesome.com/v4.7.0/examples/#fixed-width).

Then, you'll also need to set the same margin as other icons use, which is done in [`style.css`](https://github.com/mozilla/pontoon/blob/1575bfbbd28e55fe72fabffa668cfa55ed04511c/pontoon/base/static/css/style.css#L951). You'll need to extend the rule to also include `#profile .menu li i.fab`:
```css
#profile .menu li i.fa,
#profile .menu li i.fab {
  ...
}
```

Reply:
Thanks @mathjazz  for feedback, I am currently implementing it. Is there any problem if I git ammend --no-edit and force push?

Reply:
No need to amend, simply commit changes separately - that way it's easier to review between review rounds.

Before pushing upstream, I'll merge all commits into one.

Reply:
@mathjazz having the fixed width, `fa-fw` for icons doesn't fix the width of the Github icon. 
I have also tried extending the same margin as other icons use include `#profile .menu li i.fab` and still it doesn't fix it. 
<img width="564" alt="Screenshot 2019-10-08 at 11 15 08" src="https://user-images.githubusercontent.com/17095461/66378998-48bdcd80-e9bd-11e9-8c9e-cc197c9c8fc6.png">

When I try to set a custom `fab-gh` class in `style.css` for the github icon it works
<img width="591" alt="Screenshot 2019-10-08 at 11 19 47" src="https://user-images.githubusercontent.com/17095461/66379255-c2ee5200-e9bd-11e9-9252-a589c085b6df.png">

Reply:
@katherine95 Maybe there's some caching issue? I've tried the proposed change locally and it works, which is expected - because all the icons should use the same styling.

You can try to run `make build` and then `make run` again to make sure all files are updated.

Reply:
Yes, it has now worked after running `make build`. @mathjazz I would like to test for the changes I am making in `frontend/src/core/user/components/UserMenu.js` component. How can I access/reproduce the User interface powered by that component?

Reply:
This files power our new Translate UI, which we're now rolling out to users.

There's one command you need to run within `make shell` to turn on the new UI:
https://github.com/mozilla/pontoon/blob/master/frontend/README.md#this-feature-is-behind-a-switch

Reply:
I have followed the steps in the README file. Where can I access the new Translate UI? @mathjazz

Reply:
Just open any translate view, e.g. http://localhost:8000/projects/tutorial/playground/.

The changes compared to the old UI are very subtle, the most noticeable one being the new navigation bar. https://ffp4g1ylyit3jdyti1hqcvtb-wpengine.netdna-ssl.com/l10n/files/2019/09/new.png

Reply:
Got it thanks

Reply:
@mathjazz I am facing some blocker, after activating the new UI using `./manage.py waffle_flag translate_next --everyone --create` accessing any of the tranlate languages doesn't load the page, for example `http://localhost:8000/translate/af/tutorial/all-resources/?status=translated` can't load, this is a sample screenshot.
<img width="1422" alt="Screenshot 2019-10-14 at 13 30 32" src="https://user-images.githubusercontent.com/17095461/66745298-44932380-ee87-11e9-87ca-a8a4a4579ee9.png">

Reply:
You should see an error in the DevTools Console if you scroll up. Could you paste the full error here?

Reply:
It turns out we have a bug!
https://bugzilla.mozilla.org/show_bug.cgi?id=1588565

Reply:
I see. I will just push my current changes as of now. Is that okay? @mathjazz